modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/intent_to_file_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class IntentToFileController < ApplicationController
+      INTENT_TO_FILE_TYPES = %w[compensation pension survivor].freeze
+
+      before_action :check_feature_toggle
+      before_action :validate_file_type, only: %i[show create]
+      before_action :validate_poa_code, only: %i[show create]
```

Comment:
Also a nit. At some level it feels more appropriate to reflect a `403` before a `400` due to an unsupported param/value. I.e., switching the order these two `before_action`s are registered. It would take slightly longer to respond in the case that the person is authorized + passed bad args, but I think the rationale above overrules that.